docker-compose.yml

```diff
@@ -1,49 +1,102 @@
-# Docker Compose file Reference (https://docs.docker.com/compose/compose-file/)
-
 version: '3.4'
 
-# Define services
 services:
-#### Uncomment section bellow if you want to start the latest go-ipfs instance
-  cas:
-    # Configuration for building the docker image for the service
-    build:
-      context: . # Use an image built from the specified dockerfile in the current directory.
-      dockerfile: Dockerfile
-    restart: unless-stopped
-    environment: # Pass environment variables to the service
+
+  cas_api:
+    build: .
+    environment:
       - NODE_ENV=dev
       - APP_MODE=server
       - APP_PORT=8081
-      - ANCHOR_SCHEDULE_EXPRESSION=0 0 */1 * * *
-      - IPFS_API_URL=ipfs:5001
+
+      - ANCHOR_CONTROLLER_ENABLED=false
+      - ANCHOR_EXPIRATION_PERIOD=0
+      - ANCHOR_SCHEDULE_EXPRESSION=0/1 * * * ? *
+
       - BLOCKCHAIN_CONNECTOR=ethereum
-      - ETH_NETWORK=ganache
+      - CERAMIC_API_URL=http://host.docker.internal:7007
       - ETH_GAS_LIMIT=4712388
       - ETH_GAS_PRICE=100000000000
-      - ETH_RPC_HOST=http://ganache
-      - ETH_RPC_PORT=8545
-      - ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
+      - ETH_NETWORK=ropsten
+      - ETH_OVERRIDE_GAS_CONFIG=false
+      - ETH_WALLET_PK=0x16dd0990d19001c50eeea6d32e8fdeef40d3945962caf18c18c3930baa5a6ec9
+
+      - IPFS_API_URL=ipfs:5001
+      - IPFS_API_TIMEOUT=10000
+
       - LOG_LEVEL=debug
       - LOG_TO_FILES=true
-      - TYPEORM_CONNECTION=sqlite
-      - TYPEORM_HOST=localhost
+      - LOG_PATH=/usr/local/var/log
+      - MERKLE_DEPTH_LIMIT=0
+
+      - TYPEORM_CONNECTION=postgres
+      - TYPEORM_DATABASE=anchor_db
+      - TYPEORM_HOST=database
```

Comment:
What does this `host` field represent?  Not a DNS URL I guess?

Reply:
Docker has it's own internal networking so `database` in this case points to the database container